integreat_cms/firebase_api/firebase_api_client.py

```diff
@@ -135,6 +135,7 @@ def send_pn(self, pnt: PushNotificationTranslation, region: Region) -> bool:
                 "android": {
                     "ttl": "86400s",
                 },
+                "fcm_options": {"analytics_label": region.slug},
```

Comment:
Why are we not using the full topic here as well? That would immediately provide the necessary details?
```suggestion
                "fcm_options": {"analytics_label": f"{region.slug}-{pnt.language.slug}-{self.push_notification.channel}",
```

Reply:
It seems to me that this would increase complexity without providing any real benefit, as the channel remains unchanged and the language should not impact the analytics. If the business requirements call for analytics based on the selected language, we can adjust accordingly. However, unless that is the case, I’d prefer to keep it as simple as possible. What do you think, @svenseeberg?

Reply:
AFAICT it would be very useful to have subscribers per language, not only per region.

Reply:
Hey @svenseeberg, could you provide concrete tickets or data to support the assumption that this requirement is imminent? @osmers, is there currently a specific need for tracking the received push notifications per region and language?

I’m not trying to be overly rigid, but I believe it’s important to challenge our assumptions and ensure we are focusing on current, user-centered needs. Implementing this change now, without clear evidence of its necessity, might add unnecessary complexity to our codebase.

Looking forward to your thoughts! ☺️

Reply:
@deen13 municipalities often ask for numbers of subscribed users so they can have an idea of how many people they are reaching with their push notifications - this information let's them decide whether to continue their work, increase awareness of the app, put work into reaching specific language groups. So I agree with @svenseeberg that separating the number by languages would be something the municipalities will ask for/are already asking for.

Reply:
If this suggestion is to be implemented, is `-` the appropriate separator here? Do we have region slugs containing this (or alternative separator that might be suggested)?

Reply:
Hey @osmers, thanks for the clarification! I've added the language slug to the analytics label.

Reply:
> @deen13 municipalities often ask for numbers of subscribed users so they can have an idea of how many people they are reaching with their push notifications - this information let's them decide whether to continue their work, increase awareness of the app, put work into reaching specific language groups.

@osmers I've already implemented a text display showing the average number of users reached within the news tab. I propose adding a new section to the analytics area featuring a line chart that displays the number of users reached per language and day. This would give users a clear overview of how each language is performing, helping them decide which languages to target with the next push notifications. What do you think? ☺️

Reply:
Hi @deen13 :) 
can you show me what the already implemented part looks like? 
Not sure if showing it per day is necessary - generally we recommend sending push notifications not more than once a week. It's probably not feasible to just show it for every message? Maybe even within the message page? (Not sure if that is the best place to show it :D)
Also, generally they should send the push in every language - so I'd say them seeing the users reached per language is more an info to figure out which language groups need to be informed more about the existence of the app in general